src/main/java/com/hazelcast/aws/utility/EC2RequestSigner.java

```diff
@@ -43,24 +41,25 @@
     private static final int DATE_LENGTH = 8;
     private static final int LAST_INDEX = 8;
 
-    private final AwsConfig config;
     private final String timestamp;
 
     private String service;
     private Map<String, String> attributes;
+    private String region;
```

Comment:
If these are not mutable, let's make them final?

Reply:
Added.